File: haikan-parent/haikan-manager-web/src/main/webapp/js/service/visitorService.js
app.service('visitorService',function($http){
	
	//分页查询
	this.findPage=function(pageNum,pageSize){
		return $http.get('../visitor/findPage.do?pageNum='+pageNum+"&pageSize="+pageSize);
	}
	
	//批量删除
	this.dele=function(ids){
		return $http.get('../visitor/delete.do?ids='+ids);
	}
});

File: haikan-parent/haikan-manager-interface/src/main/java/com/haikan/manager/service/VisitorService.java
package com.haikan.manager.service;

import entity.PageResult;

/**
 * 访客服务层接口
 * @author Administrator
 *
 */
public interface VisitorService {

	/**
	 * 分页查询访客信息
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);
	
	/**
	 * 添加访客信息
	 * @param idCard
	 */
	public void save(String idCard);
}
